projects/classification/sensor/artificial_data/data/sensortag/jump-5min.csv

```diff
@@ -0,0 +1,1415 @@
+timestamp,x,y,z,g,raw_x,raw_y,raw_z
```

Comment:
There is another sensor/sensortag_data/ directory that is parallel with sensor/artificial_data/ We might want to rename the directories to avoid confusion